source/features/enable-file-links-in-compare-view.tsx

```diff
@@ -61,7 +61,7 @@ void features.add(__filebasename, {
 	],
 	exclude: [
```

Comment:
I just realized that this feature also doesn't make sense when the repo has been deleted but the PR is open. It appears as "unknown repository" and it's worth checking other features relying on "get head branch"